includes/shortcodes.php

```diff
@@ -106,12 +224,13 @@ function tsml_next_meetings($arguments)
     return '<h3>Types</h3><ul>' . implode($types) . '</ul>';
 });
 
-//output a react meeting finder widget https://github.com/code4recovery/tsml-ui
-function tsml_ui()
+/*output a react meeting finder widget https://github.com/code4recovery/tsml-ui
+ *examples: [tsml_ui], with parameters: [tsml_ui key="type" value="women"] */
```

Comment:
could the format for this be `[tsml_ui type="women,closed"]`

also if the comment format could be consistent with the other functions (above the definition, with a double slash in front), thanks

Reply:
Could not make the suggested format work.  I was able to pass the two types as a single parameter, but unable to get the formatting right to generate the tsml_react_config object. 
Commenting format fixed.

Reply:
`['women', 'closed']` should filter to meetings that are both women and closed in TSML UI

Reply:
Done

Reply:
looks like the pattern you're suggesting is `[tsml_ui key="type" first_value="women" second_value="closed" third_value=""]` but i still really think it should be `[tsml_ui type="women,closed"]`

Reply:
```
    //get parameters
    $arguments = shortcode_atts(['type' => ''], $arguments, 'tsml_ui');
    $defaults = [
        'type' => array_map('sanitize_title', explode(',', $arguments['type']))
    ];
```

Reply:
I think the checked in code is much clearer, and most importanly: it work's...

Reply:
wow, i'm sorry you feel that way. looks like we're at an impasse then

Reply:
Yea! I'm sorry too...

Reply:
to be fair, `[tsm_ui key="type" value="women,closed"]` would be the first thing I'd try. The other feels a little obtuse. Feels worth a few more lines of code to follow how the shortcode would be naturally used, just going off other's I work with.

Reply:
BUT, hey, maybe fodder for the next standup?

Reply:
thanks @gkovats but how would you handle it if they asked for more filters? my approach allows room for more, eg

`[tsml_ui type="lgbtq" weekday="tuesday" region="usa"]`

it seems like a code smell to use two arguments that need to work in combination when one is more direct

Reply:
oh, yeah, when I saw the shortcode figured `key="type"` was a standard. ^ that'd be even more familar, key value pairs. Sorry, my original was "it works" without context. Yeah, then we'd just have accepted shortcode attributes to look.